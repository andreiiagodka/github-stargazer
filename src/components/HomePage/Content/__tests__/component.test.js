import { shallow } from 'enzyme'

import ContentComponent from '..'

describe('<ContentComponent />', () => {
  it('renders ListGroup', () => {
    const repositories = [
      {
        id: 'id',
        full_name: 'full_name',
        stargazers_count: 1 
      }
    ]

    const props = { repositories: repositories }
    const wrapper = shallow(<ContentComponent {...props} />)

    expect(wrapper).toMatchSnapshot()
  })

  it('renders BlankComponent', () => {
    const props = { repositories: [] }
    const wrapper = shallow(<ContentComponent {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
