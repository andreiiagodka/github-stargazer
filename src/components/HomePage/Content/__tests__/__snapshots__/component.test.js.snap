// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentComponent /> renders BlankComponent 1`] = `<BlankComponent />`;

exports[`<ContentComponent /> renders ListGroup 1`] = `
<ListGroup>
  <RepositoryComponent
    id="id"
    key="id"
    name="full_name"
    stars={1}
  />
</ListGroup>
`;
